config.frontend.test.yaml

```diff
@@ -0,0 +1,202 @@
+            x-modules:
+              - path: projects/v1/enwiki.wmf.yaml
+                options: *default_options
+          /{api:sys}: *default_sys
+
+wiktionary_project: &wiktionary_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: test/test_module.yaml
+              - path: projects/v1/wiktionary.wmf.yaml
+                options: *default_options
+          /{api:sys}: *default_sys
+
+# Hacky way to parametrize RESTBase tests. TODO: Move to config?
+test:
+  content_types:
+    html: '/^text\/html; charset=utf-8; profile="https:\/\/www\.mediawiki\.org\/wiki\/Specs\/HTML\/[\d.]+"$/'
+    data-parsoid: '/^application\/json; charset=utf-8; profile="https:\/\/www\.mediawiki\.org\/wiki\/Specs\/data-parsoid/[\d.]+"$/'
+    wikitext: '/^text\/plain; charset=utf-8; profile="https:\/\/www.mediawiki.org\/wiki\/Specs\/wikitext\/[\d.]+"$/'
+
+
+# The root of the spec tree. Domains tend to share specs by referencing them
+# using YAML references.
+spec_root: &spec_root
+  title: "The RESTBase root"
+  # Some more general RESTBase info
+  x-request-filters:
+    - path: lib/security_response_header_filter.js
+    - path: lib/normalize_headers_filter.js
+
+  x-sub-request-filters:
+    - type: default
+      name: http
+      options:
+        allow:
+          - pattern: /^https?:\/\/[a-zA-Z0-9\.]+\/w\/api\.php/
+            forward_headers: true
+          - pattern: /^https?:\/\/parsoid-beta.wmflabs.org.+/
+            forward_headers: true
+          # Need to forward cookie to backend RESTBase
+          - pattern: /^https?:\/\/localhost:7231.+/
+            forward_headers: true
+          - pattern: /^https?:\/\//
+  paths:
+    /{domain:test.wikipedia.org}: *wikipedia_project
+    # The order is important for tests.
+    # Redirect tests require en.wiki being not the first wiki in the list.
+    /{domain:en.wikipedia.org}: *enwiki_project
+    /{domain:ru.wikipedia.org}: *wikipedia_project
+    /{domain:de.wikipedia.org}: *wikipedia_project
+    /{domain:test2.wikipedia.org}: *wikipedia_project
+    /{domain:commons.wikimedia.org}: *default_project
+
+    # labs, used for most tests
+    /{domain:en.wikipedia.beta.wmflabs.org}: *wikipedia_project
+    # Serbian wiki in beta for language variants tests
+    /{domain:sr.wikipedia.beta.wmflabs.org}: *wikipedia_project
+
+    # For security testing we rely on mocks, so it's OK to use French wiki.
+    /{domain:fr.wikipedia.org}:
+      <<: *wikipedia_project
+      x-route-filters:
+        - path: ./lib/mediawiki_auth_filter.js
+          options:
+            permissions:
+              - read
+    # global domain
+    /{domain:wikimedia.org}: *wikimedia_project
+
+    # Wiktionary has some specific endpoints
+    /{domain:en.wiktionary.org}: *wiktionary_project
+
+
+
+# Finally, a standard service-runner config.
+info:
+  name: restbase
+
+services:
+  - name: restbase
+    module: hyperswitch
+    conf:
+      port: 7232
```

Comment:
7232 is used by AQS, so let's use 7233 to keep our sanity (this is also most likely the port to be used in production)

Reply:
Done.